tests/core/math/test_plane.h

```diff
@@ -98,7 +100,11 @@ TEST_CASE("[Plane] Plane-point operations") {
 			y_facing_plane.get_any_perpendicular_normal().is_equal_approx(Vector3(1, 0, 0)),
 			"get_any_perpindicular_normal() should return the expected result.");
 
-	// TODO distance_to()
+	CHECK(
+			Math::is_equal_approx((double)y_facing_plane.distance_to(origin_point), -4.0));
+
+	CHECK(
+			Math::is_equal_approx((double)plane.distance_to(plane.get_center()), 0.0));
```

Comment:
We can remove the line breaks, they actually waste horizontal space here:

```suggestion
	CHECK(Math::is_equal_approx((double)y_facing_plane.distance_to(origin_point), -4.0));

	CHECK(Math::is_equal_approx((double)plane.distance_to(plane.get_center()), 0.0));
```